config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -53,7 +53,7 @@ presubmits:
         - name: KUBE_VERIFY_GIT_BRANCH
           value: master
         - name: REPO_DIR
-          value: /workspace/k8s.io/kubernetes
+          value: /home/prow/go/src/k8s.io/kubernetes
```

Comment:
prow checks out to `/home/prow/go/src/<repo>` natively in pod-utils. this other path is relying on symlink hacks that will probably cause us problems.